pkg/oc/cli/image/info/info.go

```diff
@@ -11,6 +11,8 @@ import (
 	"text/tabwriter"
 	"time"
 
+	"github.com/openshift/origin/pkg/oc/cli/image/workqueue"
```

Comment:
You're messing our imports :disappointed:

Reply:
Will fix, I have more improvements coming as I add versioning